### PR TITLE
visualize decision trees

### PR DESCRIPTION
**Pull request by karina-usmanova**
Before submitting a pull request, please do the following steps:

1. Read instructions for contributors [here](https://tech.yandex.com/catboost/doc/dg/concepts/development-and-contributions-docpage/).
2. Run `ya make` in catboost folder to make sure the code builds.
3. Add tests that test your change.
4. Run tests using `ya make -t -A` command.
5. If you haven't already, complete the CLA.

### CODE REVIEW

File: catboost/libs/algo/tree_print.cpp [hunk 1]
```diff
@@ -1,26 +1,28 @@
 #include "tree_print.h"
 
+
```
Comment (ek-ak) [inline catboost/libs/algo/tree_print.cpp]: extra blank line

File: catboost/libs/algo/tree_print.h [hunk 2]
```diff
@@ -16,10 +19,18 @@ namespace NCB {
 
 
 TString BuildFeatureDescription(
-    const NCB::TFeaturesLayout& featuresLayout,
+    const TMaybe<NCB::TFeaturesLayout>& featuresLayout,
     const int internalFeatureIdx,
     EFeatureType type);
+
+//TString BuildFeatureDescription(const TMaybe<NCB::TFeaturesLayout>& featuresLayout, const int internalFeatureIdx, EFeatureType type);
```
Comment (ek-ak) [inline catboost/libs/algo/tree_print.h]: unnecessarily comment line

File: catboost/libs/model/model.cpp [hunk 3]
```diff
@@ -30,6 +30,7 @@
 #include <util/generic/xrange.h>
 #include <util/generic/ylimits.h>
 #include <util/string/builder.h>
+#include <util/string/cast.h>
```
Comment (ek-ak) [inline catboost/libs/model/model.cpp]: unnecessarily  line

File: catboost/libs/model/model.h [hunk 4]
```diff
@@ -814,6 +814,7 @@ struct TFullModel {
                 ObliviousTrees.CatFeatures);
         }
     }
+
```
Comment (ek-ak) [inline catboost/libs/model/model.h]: unnecessarily line

File: catboost/python-package/catboost/_catboost.pyx [hunk 5]
```diff
@@ -482,12 +482,47 @@ cdef extern from "catboost/libs/model/model.h":
         TVector[float] Borders
         TString FeatureId
 
+
```
Comment (ek-ak) [inline catboost/python-package/catboost/_catboost.pyx]: extra blank line

File: catboost/python-package/catboost/_catboost.pyx [hunk 6]
```diff
@@ -482,12 +482,47 @@ cdef extern from "catboost/libs/model/model.h":
...
+        pass
+
+    cdef cppclass TModelCtrSplit:
+        TModelCtr Ctr
+        float Border
+
+    cdef cppclass TOneHotSplit:
+        int CatFeatureIdx
+        int Value
+
```
Comment (ek-ak) [inline catboost/python-package/catboost/_catboost.pyx]: extra blank line

**Comment by kizill** [comment]
@arcadia-devtools Ship it!

**Comment by kizill** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by arcadia-devtools** [comment]
@kizill, internal review request updated: [813100](https://a.yandex-team.ru/review/813100)

**Comment by arcadia-devtools** [comment]
Check [813100](https://a.yandex-team.ru/review/813100) passed

**Pull request merged**